serverless.yaml

```diff
@@ -959,6 +959,13 @@ resources:
       FHIRBinaryBucket:
         Description: S3 Bucket for storing Binary Objects
         Value: !Ref FHIRBinaryBucket
+
+      FhirWorksAlarmSNSTopic:
```

Comment:
Can you output the Condition value so that you know whether to enable the alarm action and configure this topic?

Reply:
I think the `FhirWorksAlarmSNSTopic` is provisioned irrespective of `isAlarmSubscriptionEnabled` condition value. 
And `isAlarmSubscriptionEnabled` value depends up `pager_duty_integration_key` value that we pass through `terraform-aws-fhir-works` module.

Thus alarm action is enabled when `pager_duty_integration_key` != "". So do we need to output that condition?

Reply:
Correct. My request was to output the Condition so that it could be imported into the auditlogmover Serverless configuration, and not have to replicate the Condition using the env var.